tasks/am-devtools.yml

```diff
@@ -12,6 +12,45 @@
     - "graphviz"
     - "libgraphviz-dev"
     - "python-pygraphviz"
+  when:
+    - ansible_distribution == "Ubuntu"
+
+- name: "Install am-devtools package dependencies (RH/CentOS)"
+  yum:
+    pkg: "{{ item }}"
+    state: "latest"
+  with_items:
+    - "rubygems"
+    - "ruby-devel"
+    - "graphviz"
+    - "graphviz-devel"
+    - "graphviz-python"
+    - "python2-pkgconfig"
+  when:
+    - ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
```

Comment:
We can also us ansible_os_family,  and use only one condition, as we do here:

use https://github.com/artefactual-labs/ansible-archivematica-src/blob/7f56580b66f53647be562f0c574f870b47816f54/tasks/common.yml#L11